doc/man/bitcoin-cli.1

```diff
@@ -1,4 +1,3 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
```

Comment:
Why did you ignore (and remove) this warning? You shouldn't edit the man pages directly, they're generated automatically with a script before releases. See the [release process](https://github.com/bitcoin/bitcoin/blob/master/doc/release-process.md).

Reply:
My bad, @laanwj. Thanks for the response, that's good to know. The man pages seemed really out of date with the current state (but not the current release), so I assumed they were generated as a one-off at some point. Will withdraw the PR.

Reply:
FWIW I completely agree with the intent of this PR, but any improvements to man page generation would have to be scripted, to avoid having to do manual changes every time. There used to be fully manually-maintained man pages but no one really kept them up to date for years, so unless someone is now really enthusiastic in doing that, the scripted approach is preferable.